src/seedu/addressbook/data/person/Person.java

```diff
@@ -88,4 +88,8 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person p) {
+        return getName().toString().compareTo(p.getName().toString());
```

Comment:
Good job Hansel! I can see the effort of adding a new sort method that sort according to name.